ludwig/constants.py

```diff
@@ -53,7 +53,7 @@
 SEQUENCE_SOFTMAX_CROSS_ENTROPY = "sequence_softmax_cross_entropy"
 SOFTMAX_CROSS_ENTROPY = "softmax_cross_entropy"
 SIGMOID_CROSS_ENTROPY = "sigmoid_cross_entropy"
-BINARY_WEIGHTED_CROSS_ENTROPY = "binary_weighted_cross_entropy"
+BINARY_WEIGHTED_CROSS_ENTROPY_LOSS = "binary_weighted_cross_entropy_loss"
```

Comment:
Why is `BINARY_WEIGHTED_CROSS_ENTROPY` suffixed with `_LOSS`, when none of the other cross_entropy metrics are (i.e. `SOFTMAX_CROSS_ENTROPY`, `SIGMOID_CROSS_ENTROPY`)?